src/containers/offer-details/offer-general-info/constants.tsx

```diff
@@ -0,0 +1,57 @@
+import CategoryIcon from '@mui/icons-material/Category'
+import BarChartIcon from '@mui/icons-material/BarChart'
+import LanguageIcon from '@mui/icons-material/Language'
+import SellIcon from '@mui/icons-material/Sell'
+import DoneIcon from '@mui/icons-material/Done'
+
+import IconTitleDescription from '~/components/icon-title-description/IconTitleDescription'
+
+import { Offer } from '~/types'
+
+const icon = <DoneIcon fontSize='small' sx={{ color: 'basic.orientalHerbs' }} />
+
+const style = {
+  container: {
+    display: 'flex',
+    columnGap: '5px'
+  },
+  icon: {
+    display: 'flex',
+    alignItems: 'center'
+  },
+  title: {
+    typography: { xs: 'body2', sm: 'body1' }
+  }
+}
+
+export const offerGeneralInfoMock = (offer: Offer) => {
```

Comment:
Why do you use the mock word? if we have offers created and working properly already

Reply:
it is not mock for offer itself, it is mock with icon, title, descr for generalInfo cards

Reply:
fixed